resinhup.py

```diff
@@ -117,6 +117,13 @@ def main():
         log.error("Resinhup is not supported for the resinOS version requested.")
```

Comment:
I think this expression suffers from the same problem as pointed out below ("2.0.0-beta6"). "2.0.0-beta6" is not a valid version string supported by distutils.version which this program uses. One solution is to stop using the non-standard versions. The other would be to sanitize these versions and convert them to something that's supported. Comments?